dms_action/views/dms_file.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 CreuBlanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="dms_file_form_view">
+        <field name="name">dms.file.form (in dms_action)</field>
+        <field name="model">dms.file</field>
+        <field name="inherit_id" ref="dms.view_dms_file_form" />
+        <field name="arch" type="xml">
+            <header position="inside">
+                <button
+                    name="action_execute_action"
+                    type="object"
+                    icon="fa-play"
+                    string="Play action"
+                />
+            </header>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_dms_file_kanban">
+        <field name="name">dms.file.form (in dms_action)</field>
+        <field name="model">dms.file</field>
+        <field name="inherit_id" ref="dms.view_dms_file_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('o_kanban_image')]" position="before">
+
+                <div class="mk_file_kanban_actions">
+                    <div class="mk_file_kanban_actions_wrapper">
+                        <a
+                            type="object"
+                            name="action_execute_action"
+                            role="button"
+                            class="btn btn-sm btn-outline-primary mk_directory_kanban_directories"
+                            title="Play action"
```

Comment:
Some possibilities about it:
- A: Hide the button when there are no actions available (it is strange to press and that the wizard is empty).
- B: Place the button just before the block button (in this way that block would not be added to the left of the file image).
- C: Define an icon for each action, and place them directly in the kanban view before the lock button (this would avoid the wizard).

IMO option C is the most optimal.

Reply:
Did the change on the kanban (really hard, by the way :rofl: ). If you like it, I will do something similar on the form with the header and remove the wizard. WDYT? @victoralmau

Reply:
I think it's better this way in the kanban view, if you can change it in the form view to remove wizard it would be perfect.